funsite/templates/lms/register.html

```diff
@@ -152,6 +154,12 @@ <h2 class="color-regal-blue">
                             link_start='<a href="{url}"  tabindex="-1">'.format(url=honor_code_path),
                             link_end='</a>')}</label>
                     </div>
+
+                    <div class="form-group" id="field-newsletter">
+                        <input id="newsletter" type="checkbox" value="true" />
+                        <label for="newsletter">${_('I would like to receive updates from {platform_name}').format(
```

Comment:
You also have to add the translation in your PR, they are not managed in another service. You will found them in `funsite/locale`

And the exact expected translation is 
```
J'accepte de recevoir vos emails sur l’offre de formations professionnelles payantes FUN et confirme avoir pris connaissance des [Conditions générales de la newsletter formation professionnelle](https://www.fun-mooc.fr/fr/cgu/cgu-newsletter-formapro/).
```

That also means a link is present in the sentence.